<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>911</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>961</width>
      <height>811</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(/Users/Isaac/Desktop/FinalProject/Sudoku/backtracking/backtracking/);</string>
    </property>
    <widget class="QGroupBox" name="menubox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>541</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(214, 214, 214,0.8)</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="xsudoku">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 117, 71);
background:rgb(234, 232, 236,0.1);</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>111</width>
        <height>4</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>4</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #75a3a3;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="titleLbl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #006666;
background:rgb(214, 214, 214,0.7);</string>
      </property>
      <property name="text">
       <string>  BackTracking Algorithms Exploration</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Hello">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(227, 128, 7);
background: #F0F8FF;
border: 1px solid rgb(255, 85, 0);
border-radius: 15px;
</string>
      </property>
      <property name="text">
       <string>Hello!</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>351</width>
        <height>2</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="xsudoku_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(76, 78, 83);
background:rgb(241, 255, 246,0.3);</string>
      </property>
      <property name="text">
       <string>@SuDOkU~ </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="solve">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>solve</string>
      </property>
     </widget>
     <zorder>line</zorder>
     <zorder>titleLbl</zorder>
     <zorder>Hello</zorder>
     <zorder>line_2</zorder>
     <zorder>xsudoku_2</zorder>
     <zorder>xsudoku</zorder>
     <zorder>solve</zorder>
    </widget>
    <widget class="QGroupBox" name="controlpanel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>181</width>
       <height>591</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
background: rgb(230, 230, 230,0.5);
QPushButton#erasebtn:hover{background:green;\n}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="constraint">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 102, 0);
background-color: rgb(0, 170, 255);
</string>
      </property>
      <property name="text">
       <string>Show Constraints</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="numberbox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>171</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(229, 227, 221,0.5);</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>470</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(78, 199, 255);
background: #F0F8FF;


</string>
      </property>
      <property name="text">
       <string>RE-Start</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="newGameBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>181</width>
        <height>151</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(85, 170, 255);
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="yesNew">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 40, 2);
background:rgb(249, 249, 249);</string>
       </property>
       <property name="text">
        <string>SuRe!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="noNew">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(245, 245, 245);
color:rgb(255, 55, 37);</string>
       </property>
       <property name="text">
        <string>No..</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ArE yOu SuRe tO 
 PLaY A neW GaMe
?</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>noNew</zorder>
      <zorder>yesNew</zorder>
      <zorder>label_6</zorder>
     </widget>
     <widget class="QPushButton" name="CancelNum">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(151, 150, 152);</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="patterns">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 102, 0);
background-color: rgb(0, 170, 255);
</string>
      </property>
      <property name="text">
       <string>patterns</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="patternbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="patternLayout"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="nishio">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 102, 0);
background-color: rgb(0, 170, 255);
</string>
      </property>
      <property name="text">
       <string>nishio</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="nishiobox">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>400</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="nishiobtnGrid"/>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Board">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>521</width>
       <height>411</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 255, 255);


</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="choice">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(20, 201, 255,0.5);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="q1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>grid size:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="q1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>game mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="selectedsize">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="selectedmode">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="nine">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>9 x 9</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sixteen">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>71</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>16 x 16</string>
       </property>
      </widget>
      <widget class="QPushButton" name="heurimode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>71</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Heuri</string>
       </property>
      </widget>
      <widget class="QPushButton" name="algomode">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>71</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Algo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectionOk">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>112</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QPushButton" name="jigsawmode">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>71</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>jigsaw</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(78, 199, 255);
background: #F0F8FF;


</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Info">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>100</y>
       <width>111</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
